<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warenliste</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
    <item row="0" column="0">
     <widget class="QTableView" name="inventoryView"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>New Item</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="leditID">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>ID-Nr.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leditDescription">
         <property name="placeholderText">
          <string>Bezeichnung</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leditCosts">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>0 â‚¬</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>leditID</tabstop>
  <tabstop>leditDescription</tabstop>
  <tabstop>leditCosts</tabstop>
  <tabstop>btnAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
